client/src/pages/nearest-hospital/NearestHospital.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from "react";
+import NrMap from "./NrMap.js";
+import "./NrMap.css";
+
+class NearestHospital extends Component {
+  render() {
+    return (
+      <div>
+        <br />
+        <h5 style={{ display: "flex", justifyContent: "center" }}>
```

Comment:
please avoid inline style props and .css files
I realise I also started like that in the hackathon but that was just to get something working fast
so now that we have StyledComponents, try to use that as much as possible :)